README.md

```diff
@@ -14,6 +14,7 @@ possible!
 
 | Deadline | Scholarship | Results Released | Eligibility |
 |----------|-------------|------------------|------------|
+| May 08 | [GHC Conference Scholarship] (http://ghc.anitaborg.org/2017-student-academic/2017-scholarships/) | | Full time student or faculty scholar at an accredited university. |
 
 No scholarships are currently active. Please come back and check later!
```

Comment:
![screen shot 2017-02-21 at 1 55 45 pm](https://cloud.githubusercontent.com/assets/2482343/23179895/8347edf6-f83d-11e6-9d7b-e54283fa5563.png)
You'd be right on that.